react/components/Product/Product.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl'
+import { ProductImage, ProductPrice } from 'vtex.order-details'
+import { CurrencyContext } from '../../OrderPlaced'
+import { intlMessage } from '../../utils'
+
+const Product = ({ productInfo, intl }) => (
+  <CurrencyContext.Consumer>
+    {currency => (
+      <div className="flex justify-between mv4">
+        <div className="flex items-center">
+          <ProductImage
+            url={productInfo.imageUrl}
+            alt={productInfo.name}
+            className="w3 mr5"
+          />
+          <div className="flex flex-column items-between h-100">
+            <p
+              href={productInfo.detailUrl}
+              className="t-body c-muted-1"
+              target="_blank"
+            >
+              {productInfo.name}
+              {productInfo.measurementUnit !== 'un' && (
```

Comment:
e se for igual a `un`?

Reply:
Se for igual a `un`, eu não preciso renderizar esse texto menor em baixo do nome do produto. Basta o texto que aparece na parte inferior do componente, que de qualquer jeito vai dizer `X unidades`